src/lib.rs

```diff
@@ -254,6 +255,13 @@ pub trait Config: 'static {
         + bytemuck::CheckedBitPattern
         + bytemuck::Zeroable;
 
+    /// How GGRS should predict the next input for a player when their input hasn't arrived yet.
+    ///
+    /// [RepeatLastInputPredictor] is a good default.
+    ///
+    /// See documentation for [InputPredictor] for more information.
+    type InputPredictor: InputPredictor<Self::Input>;
```

Comment:
Note to self: I also need to update the Sync+Send variant of the Config struct to include this param.